charon/utils/logs.py

```diff
@@ -72,3 +72,16 @@ def set_logging(name="charon", level=logging.DEBUG, handler=None):
     logger = logging.getLogger('charon')
     for hdlr in list(logger.handlers):  # make a copy so it doesn't change
         hdlr.setFormatter(formatter)
+
+
+def getLogger(name: str) -> logging:
+    error_log = "./errors.log"
+    handler = logging.FileHandler(error_log)
+    formatter = logging.Formatter('%(asctime)s - %(levelname)s - %(message)s')
+    handler.setFormatter(formatter)
+    handler.setLevel(logging.ERROR)
+    handler.setLevel(logging.WARN)
```

Comment:
Why setLevel twice here?

Reply:
@ligangty Removed the duplicated unuseful level setting.